### ISSUE TITLE
Support for other gelsight-based sensors

### ISSUE THREAD

**Issue by keiohta** [Genesis-Embodied-AI/DiffTactile#1]
Hi authors, I'm very impressed with your work and I'm interested in using your codebase with a [GelSight Mini sensor](https://www.gelsight.com/gelsightmini/). Would it be possible to adapt the code for this device? If so, could you offer some pointers on how to get started? Thanks!

**Comment by blakery-star** [comment]
Yes, our code can support the GelSight Mini sensor. But you need to modify something to adapt it. Here is the list: 1. mesh of the sensor surface 2.camera model (we apply fisheye camera model in our code). Then you can apply the method of system identification in our paper to infer the physical parameters of the sensor in the simulation.

**Comment by Rancho-zhao** [comment]
Hi, I would like to ask about the specific type of the tactile sensor used in the paper. Is it a variant of [DenseTact](https://ieeexplore.ieee.org/abstract/document/9811966)?

**Comment by blakery-star** [comment]
Thanks for your question, you can read our appendix A.6.1 for more details about the tactile sensor.

**Comment by Rancho-zhao** [comment]
> Thanks for your question, you can read our appendix A.6.1 for more details about the tactile sensor.

Thank you for your reply. Actually, my question is that the GelSight sensor used in the paper seems to be different from other normal GelSight sensors with a flat surface. So the sensor you used is designed by yourselves, or refers to other papers?

**Comment by Si-Lynnn** [comment]
Hi, thanks for the question! Our sensor was designed and manufactured in the lab by one of our collaborators. Unfortunately, the design is not published as a paper yet. But our simulator can be used for sensors with flat surface, but you might need to generate the mesh model of the sensor elastomer and use the right camera model.
